iceoryx_posh/include/iceoryx_posh/internal/roudi/process_manager.hpp

```diff
@@ -158,14 +158,17 @@ class ProcessManager : public ProcessManagerInterface
 
     /// @brief Removes the process from the managed client process list, identified by its id.
     /// @param [in] name The process name which should be removed.
+    /// @param [in] sendAckToProcess Informs process that the termination messsage was received
     /// @return Returns true if the process was found and removed from the internal list.
-    bool searchForProcessAndRemoveIt(const ProcessName_t& name) noexcept;
+    bool searchForProcessAndRemoveIt(const ProcessName_t& name, const bool sendAckToProcess) noexcept;
```

Comment:
Please no bool. Do you still know in half a year what `searchForProcessAndRemoveIt(foo, false);` means?

Please create an enum for this.